components/style/traversal.rs

```diff
@@ -201,6 +211,7 @@ pub fn recalc_style_at<'a, N, C>(context: &'a C,
     let mut bf = take_thread_local_bloom_filter(parent_opt, root, context.shared_context());
 
     let nonincremental_layout = opts::get().nonincremental_layout;
+    let mut restyle_result = RestyleResult::Continue;
     if nonincremental_layout || node.is_dirty() {
```

Comment:
I wonder if this might be clearer as:

    if !nonincremental_layout && !node.is_dirty() {
        return RestyleResult::Continue
    }

Then you can remove a level of indentation below.